### PR TITLE
3.1 - Multiple Flash messages

### PR DESCRIPTION
**Pull request by HavokInspiration**
Document the addition of the append behavior of the Flash messages.

Ref cakephp/cakephp#6161

### CODE REVIEW

**Comment by antograssiot** [comment]
@HavokInspiration Nice work on the PR and the doc :+1:
Could you please just add the new files to the toc in ``{lang}/apendices.rst`` ?

**Comment by HavokInspiration** [comment]
Done.
I took the liberty of deleting the "3.0 is still under development" part, since it's not true anymore.
I also did it for languages which does not have a translation.

**Comment by cake17** [comment]
@HavokInspiration great work on this ! :+1:

File: en/controllers/components/flash.rst [hunk 1]
```diff
@@ -32,6 +32,12 @@ use the ``set()`` method::
 
     $this->Flash->set('This is a message');
 
+
+.. note::
+
+    New in 3.1 : Flash messages now stack. Successive calls to ``set()`` or ``__call()`` with
```
Comment (markstory) [inline en/controllers/components/flash.rst]: Normally we use `.. versionadded:: 3.1`

Comment (HavokInspiration) [inline en/controllers/components/flash.rst]: I'll change them

**Comment by HavokInspiration** [comment]
Note to self : add a note about how to default to the old behavior

**Comment by HavokInspiration** [comment]
Done :)

**Pull request merged**